src/connectors/tidal.js

```diff
@@ -1,42 +1,31 @@
 'use strict';
 
-let useShortTrackNames = false;
```

Comment:
One of the big changes is that Tidal seems to no longer provide longer and shorter track names. If this is confirmed and the feature needs removing, I can add another commit to delete the option from the options page.